### DISCUSSION TITLE
feature request: domain pricing / remaining days calc

### DISCUSSION THREAD

**Discussion by muhammetdemirel** [zmh-program/next-whois#20] [Ideas]
- Remaining days calculation 
- Bulk domain name query, and if it is vacant, the ability to show and direct the price list by reference. 
- Moz statistics, da, pa, spam score
- DNS information of the domain name, a record mx record etc. 
- Domain age, such as 1 year and 13 days.

**Comment by zhushiyo** [comment]
I agree with him

**Comment by muhammetdemirel** [comment]
Example: www.whois.tv.tr

**Comment by zmh-program** [comment]
already on todo list, will subsequently support the design in terms of domain pricing

**Comment by zmh-program** [comment]
#32
